ui/pages/reinscription.jsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Button, Center, Container, Heading, Link, Stack, Text } from "@chakra-ui/react";
+import React from "react";
+
+import { Page } from "../components";
+import { NAVIGATION_PAGES } from "../common/constants/navigationPages";
+import { NotFound } from "../theme/components/icons";
+import Head from "next/head";
+import NavLink from "next/link";
+import { Support } from "../theme/components/icons/Support.js";
+
+const Reinscription = () => {
+  const title = "Réinscription";
+  return (
+    <Page>
+      <Head>
+        <title>{title}</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <Box w="100%" pt={[4, 8]} px={[1, 1, 6, 8]} color="grey.800" paddingY="4w">
+        <Container maxW="xl">
+          <Center>
+            <Box width="769px" border="1px solid" borderColor="#E3E3FD" padding="4w" marginTop="2w" marginBottom="6w">
+              <Stack alignItems="center" spacing="4w">
+                <Support />
+                <Heading fontSize="28px" fontWeight="bold">
+                  Heureux de vous revoir !
+                </Heading>
+                <Text color="grey.800" align="center" fontSize="zeta" marginBottom="2w">
+                  Le mode de connexion a changé pour renforcer la sécurité des données. <br />
+                  Afin de retrouver votre accès aux fonctionnalités du tableau de bord de l&apos;apprentissage, nous
+                  vous remercions de créer votre nouveau compte.
+                </Text>
+
+                <Button variant="secondary" href={NAVIGATION_PAGES.Inscription.path} marginTop="2w" as={NavLink}>
```

Comment:
maintenant qu'on a next qui gere le routing, ca a vocation a rester NAVIGATION_PAGES ?

Reply:
Perso jaimerai l'enlever +1 avec Sven 
Perte de lisibilité ==> Wet code is better here

Reply:
Oui ca a du sens, avant next on utilisait cette const pour retrouver nos petits mais effectivement je suis d'accord 👉🏼 https://github.com/mission-apprentissage/flux-retour-cfas/issues/2341